com.unity.formats.fbx/Editor/Sources/Recorders/FbxRecorder/FbxRecorder.cs

```diff
@@ -45,40 +45,25 @@ protected override void EndRecording(RecordingSession session)
 #endif
                 var root = ((AnimationInputSettings)aInput.settings).gameObject;
                 clip.name = "recorded_clip";
-                clip.legacy = true;
```

Comment:
why don't we need this code anymore?

Reply:
I needed it to be legacy before because the Animation component only accepts legacy animation clips. Don't need it now that I pass the clip directly to the exporter.